### PR TITLE
Integrate new data convention devs into `develop` branch

### PR DESCRIPTION
**Pull request by sbancal**
Let's integrate all new data convention devs into `develop` branch.

- [x] Trickle up data changes into above layers : DB, backend, frontend
- [x] ready to deploy on VM with docker + db auto-provisioning
- [x] Fix following issues(or PR) :
- #36
- #38
- #39
- #41
- #42
- #43
- #50
- #51
- #52
- #53
- #55
- #57
- #58
- #60
- #61
- #70
- #79
- #85
- #86

### CODE REVIEW

**Review by guilbep — APPROVED** [review]

File: Preprocessing/tst_data_quality_control.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    + ERROR when :
+      + mandatory column is not found
+      + unexpected type is found in a column
+    + WARNING when :
+      + unexpected column found
+"""
+
+#
+# TODO :
+# + check JSON files (from XLS)
```
Comment (guilbep) [inline Preprocessing/tst_data_quality_control.py]: Is it okay ? to have a. todo ?

Comment (sbancal) [reply Preprocessing/tst_data_quality_control.py]: Ths file doesn't exist anymore in the latest commit

File: backend/init_db.py [hunk 2]
```diff
@@ -85,17 +108,31 @@ def inject_exp_from_folder(exp_folder, session):
     # TODO
```
Comment (guilbep) [inline backend/init_db.py:108]: @sbancal TODO empty ?

File: frontend/src/components/ExperimentSpecifications.vue [hunk 3]
```diff
@@ -108,8 +89,10 @@
             </li>
             <li>
               <experiment-s-v
-                subject="Fiber content"
-                :values="[experiment.laminates_and_assemblies_fiber_content]"
+                subject="Fiber volume ratio"
+                :values="[
+                  experiment.laminates_and_assemblies_fiber_volume_ratio,
```
Comment (guilbep) [inline frontend/src/components/ExperimentSpecifications.vue:94]: @sbancal empty line ?

**Pull request merged**